schema/RequestSignInLinkMutation.re

```diff
@@ -8,7 +10,7 @@ let requestSignInLinkInput =
       "RequestSignInLinkInput",
       ~fields=[arg("email", ~typ=non_null(string))],
       ~coerce=email =>
-      {email}
```

Comment:
coerce not needed then?

Reply:
just to have this effect on schema, of { input: { email: String!  }  }